app/src/main/java/io/gnosis/safe/di/modules/ViewModule.kt

```diff
@@ -119,6 +122,18 @@ class ViewModule(
     @ForView
     fun providesImportOwnerKeyViewModel(provider: ViewModelProvider) = provider[OwnerSeedPhraseViewModel::class.java]
 
+    @Provides
+    @ForView
+    fun providesOwnerEnterNameViewModel(provider: ViewModelProvider) = provider[OwnerEnterNameViewModel::class.java]
+
+    @Provides
+    @ForView
+    fun providesOwnerEditNameViewModel(provider: ViewModelProvider) = provider[OwnerEditNameViewModel::class.java]
+
+    @Provides
+    @ForView
+    fun providesOwnerListViewModel(provider: ViewModelProvider) = provider[OwnerListViewModel::class.java]
```

Comment:
Are these 3 provides-methods really necessary? I removed them and everything seems to be still working.

Reply:
we need them. Otherwise view models won't be injected into fragments. You probably had generated classes lying around after you removed those provides-methods.